scripts/test_validations.py

```diff
@@ -0,0 +1,44 @@
+from nose.tools import eq_
```

Comment:
The rest of olympia uses py.test rather than nose. It would be nice to stay consistent.